RunningOrder/Shared/MainView.swift

```diff
@@ -12,6 +12,7 @@ extension MainView {
     private struct InternalView: View {
         @EnvironmentObject var spaceManager: SpaceManager
         @ObservedObject var logic: Logic
+        @State private var searchText: String = ""
```

Comment:
this seems weird as this is not related to this view at all.. why not embed it in the SearchManager ?